src/SFA.DAS.EmployerIncentives.Web/Models/PaymentStatusModel.cs

```diff
@@ -8,5 +8,14 @@ public class PaymentStatusModel
         public DateTime? PaymentDate { get; set; }
         public bool LearnerMatchNotFound { get; set; }
         public bool HasDataLock { get; set; }
+        public bool ApprenticeNotInLearning { get; set; }
+
+        public bool HasPaymentErrorStatus
```

Comment:
Could this be ShowPaymentStatus?  Since when true it renders the PaymentStatus partial?